test/.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+	"extends": "../src/.eslintrc.json",
+	"root": true,
```

Comment:
mmm, so this is root, so it wouldn't inherit upper configs, only extends one from `src` and `src` one doesn't have `root` so it would inherit upper ones, which would be the same uppers since `src` is sibling of `test` dir, correct?